packages/components/src/tabs/README.md

```diff
@@ -0,0 +1,240 @@
+# Tabs Component
+
+A wrapper component around WordPress's private [`Tabs` component](https://wordpress.github.io/gutenberg/?path=/docs/components-tabs--docs) from `@wordpress/components`.
+
+## About
```

Comment:
Wasn't sure if to call the section "Usage" or "About" or something else. We link to docs so Usage could work.